Makefile

```diff
@@ -15,7 +15,7 @@ test :
 
 .PHONY : bench
 bench:
-	@dune build @bench
+	@ODOC_BENCHMARK=true dune build @bench
```

Comment:
I wonder if a comment here would make sense. The meaning of this variable is not documented anywhere, you have to look at the `driver.mld` code.

Reply:
Good idea, done.